### DISCUSSION TITLE
Starstruck achievement not displaying properly (possibly bugged?)

### DISCUSSION THREAD

**Discussion by pmulard** [community/community#59620] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I recently achieved the second rank on the starstruck badge by having a repository get 128 stars. Though, I've noticed the bronze x2 badge does not appear on my profile. Feel free to take a look at my profile, but below is a screenshot:

![image](https://github.com/community/community/assets/45550174/1c04d1a4-e838-45d4-a38a-c4900fc45a50)

Shouldn't this have a badge similar to the pull shark achievement, or am I missing something here? [This guide](https://github.com/drknzz/GitHub-Achievements) is my reference.

**Comment by radar-blip** [comment]
Yes, it should have an oval with the x2 inside it. 

![Screenshot from 2023-06-29 02-42-02](https://github.com/community/community/assets/39196767/7a7bb395-7e80-482e-888c-231be62583ce)

**Comment by ArdeshirV** [comment]
### Maybe you need to enable visibility of your private contributions?
### Changing the visibility of your private contributions
You can choose to publicize your private contributions, which allows visitors to your profile to see private contribution counts, without further details. If you choose to hide your private contributions, visitors will only see your public contributions.
- In the top right corner of GitHub.com, click your profile photo, then click Your profile.
![](https://docs.github.com/assets/cb-18133/mw-1440/images/help/profile/top-right-avatar.webp)
- Under "Contributions & Activity", select or deselect Include private contributions on my profile.
[![](https://i.postimg.cc/PJKcfR61/Screenshot-from-2023-06-29-18-34-11.png)](https://postimg.cc/zHbp00Yf)

**Reply by pmulard** [reply]
I already have this setting toggled to allow private contributions, so it must be something else. Hmm weird. Appreciate the suggestions though.

**Comment by YasinDehfuli** [comment]
i think its a bug, send bug issue to github to fix the problem

**Comment by pmulard** [comment]
I submitted a ticket to this issue to GitHub, then a few days later my profile was updated with the correct achievement. I never received any word on the ticket, so it seems like the bug just resolves itself after a week.
